anaron/02. HTML. Table/site.by/about.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<HTML>
+
+<HEAD>
+    <meta charset="utf-8">
+    <TITLE>Пример страницы</TITLE>
+</HEAD>
+
+<BODY>
+    <H1>about</H1>
+    <h2>Меню с вложенным списком</h2>
+    <ul>
+        <li><a href="about.html">about</a></li>

```

Comment:
this li tag should be contain whole sub list, not just tag a, look carefully our slides